text/0000-refresh-method-for-router-service.md

```diff
@@ -0,0 +1,36 @@
+- Start Date: 2020-05-23
+- Relevant Team(s): Ember.js
+- RFC PR: (after opening the RFC PR, update this with a link to it and update the file name)
+- Tracking: (leave this empty)
+
+# RouterService#refresh
+
+## Summary
+
+> Add a refresh method to the router service that calls refresh on the current route.
+
+## Motivation
+
+> We want to be able to refresh the current route without relying on the send api.
+
+## Detailed design
+
+```
+class RouterService {
+    refresh() {
+        this.owner.lookup(`route:${this.currentRouteName}`).refresh();
+    }
+}
+```
+
+## How we teach this
+
+> Documentation will be added to the method.
```

Comment:
The API documentation for the method should be included in this section. It should likely also include some details around reviewing the current guides to see if any can / should be updated to leverage the new feature.